CHANGELOG.md

```diff
@@ -2,18 +2,91 @@
 
 ## Unreleased
 
+## v0.99.0
+
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) `receiver/discovery`: Update the component to emit entity events 
```

Comment:
I'd like some more clarity on the `receiver/discovery` breaking changes as far as what's required (if anything) from users.

Reply:
@dmitryax would know most about this. From the listed PRs, the `log_endpoints` option removal seems user-facing but what are the chances that users have customized receiver/discovery setup?

Reply:
From Dmitrii's comment, nothing is required of users, but the changelog comment has been simplified.